samples/meetings-context-app/csharp/MeetingContextApp/Bots/MeetingContextBot.cs

```diff
@@ -0,0 +1,88 @@
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+
+namespace MeetingContext.Bots
+{
+    public class MeetingContextBot : TeamsActivityHandler
+    {
+        public const string commandString = "Please use one of these two commands: **Meeting Context** or **Participant Context**";
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            if (turnContext.Activity.Text != null)
+            {
+                var text = turnContext.Activity.RemoveRecipientMention();
+                if (text.ToLower().Contains("participant context"))
+                {
+                    var channelDataObject = JObject.Parse(JsonConvert.SerializeObject(turnContext.Activity.ChannelData));
+
+                    var tenantId = channelDataObject.tenant["id"].ToString();
+                    var metingId = channelDataObject.meeting["id"].ToString();
+                    var participantId = turnContext.Activity.From.AadObjectId;
+
+                    // GetMeetingParticipant
+                    TeamsMeetingParticipant participantDetails = await TeamsInfo.GetMeetingParticipantAsync(turnContext, metingId, participantId, tenantId).ConfigureAwait(false);
+
+                    var formattedString = this.GetFormattedSerializeObject(participantDetails);
+
+                    await turnContext.SendActivityAsync(MessageFactory.Text(formattedString), cancellationToken);
+                }
+                else if (text.ToLower().Contains("meeting context"))
+                {
+                    MeetingInfo meetingInfo = await TeamsInfo.GetMeetingInfoAsync(turnContext);
+
+                    var formattedString = this.GetFormattedSerializeObject(meetingInfo);
+
+                    await turnContext.SendActivityAsync(MessageFactory.Text(formattedString), cancellationToken);
+                }
+                else
+                {
+                    await turnContext.SendActivityAsync(MessageFactory.Text(commandString), cancellationToken);
+                }
+
+            }
+
+        }
+
+        protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersAdded, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var welcomeText = "Hello and Welcome!";
+
+            await turnContext.SendActivityAsync(MessageFactory.Text(welcomeText), cancellationToken);
+            await turnContext.SendActivityAsync(MessageFactory.Text(commandString), cancellationToken);
+        }
+
+        private string GetFormattedSerializeObject (object obj)
+        {
+            var formattedString = "";
+            foreach (var meetingDetails in obj.GetType().GetProperties())
+            {
+                var detail = meetingDetails.GetValue(obj, null);
+                var block = $"<b>{meetingDetails.Name}:</b> <br>";
+                var temp = "";
```

Comment:
what is temp ?
Can we use some meaningful name here ?

Reply:
It is there to just hold string temporarily. Updated it as storeTemporaryFormattedString